traffic-dashboard/src/components/LoginComponent/LoginComponent.tsx

```diff
@@ -0,0 +1,70 @@
+
+import React, { useState } from "react";
+import { AppBar, Avatar, Button, Menu, MenuItem, Tooltip, Typography } from "@mui/material";
+import IconButton from '@mui/material/IconButton';
+
+import Box from '@mui/material/Box';
+import { Login } from "@mui/icons-material";
+
+type Props = {
```

Comment:
Remove the interface if it's empty pelase

Reply:
Ok i will fix this